### PR TITLE
Added Teams

### PR DESCRIPTION
**Pull request by epintos**
## Summary
- Added teams

TODO: Add pundit in next PR
TODO: Create new users by invite

### CODE REVIEW

File: app/controllers/teams_controller.rb [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  def update
+    if team.update(team_params)
+      redirect_to_edit_team(:success, t('teams.update.success'))
+    else
+      render 'new'
+    end
+  end
+
+  def destroy
+    if @team == @organization.admin_team
```
Comment (mdesanti) [inline app/controllers/teams_controller.rb]: Where is this `@team` coming from?

Comment (epintos) [inline app/controllers/teams_controller.rb]: Its wrong, shoud be `team`. This validation is only defensive, you can't do it from the app

File: spec/factories/teams.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :team do
+    organization
```
Comment (mdesanti) [inline spec/factories/teams.rb]: What is this `organization` doing?

**Comment by mdesanti** [comment]
:+1:

File: spec/factories/teams.rb (hunk 2 above)
Comment (epintos) [inline spec/factories/teams.rb]: Its the new syntax for organization create(:organization)

**Pull request merged**